packages/rich-text/src/plugins/List/transforms/toggleList.ts

```diff
@@ -0,0 +1,163 @@
+import { BLOCKS } from '@contentful/rich-text-types';
+import {
+  ELEMENT_DEFAULT,
+  findNode,
+  getAbove,
+  getNodes,
+  getPluginType,
+  isCollapsed,
+  isRangeAcrossBlocks,
+  PlateEditor,
+  setNodes,
+  TElement,
+  unwrapNodes,
+  wrapNodes,
+} from '@udecode/plate-core';
+import { ELEMENT_LIC } from '@udecode/plate-list';
+import { getListItemEntry } from '@udecode/plate-list';
+import { Editor, Node, NodeEntry, Path, Range } from 'slate';
+
+const listTypes = [BLOCKS.UL_LIST, BLOCKS.OL_LIST] as string[];
+
+const unwrapList = (editor: PlateEditor, { at }: { at?: Path } = {}) => {
```

Comment:
Is this mostly new logic or borrowed from Plate? If the latter, perhaps leave a note.

Reply:
The whole file is borrowed from Plate. I made some modifications here and there. I will add some comments.